components/tools/OmeroPy/src/omero/config.py

```diff
@@ -188,9 +190,30 @@ def version(self, id=None):
     def version_check(self):
         for k, v in self.properties(None, True):
             version = self.version(k)
-            if version != self.VERSION:
+            if version == "4.2.0":
                 self.version_fix(v, version)
 
+    def toplinks_check(self):
+        for k, v in self.properties(None, True):
+            version = self.version(k)
+            if version == "4.2.1" and v is not None:
+                for x in v.getchildren():
+                    if x.get("name") == "omero.web.ui.top_links":
+                        val = x.get("value", "")
+                        toplinks = json.loads(val)
+                        defaultlinks = [
```

Comment:
Should we rather use `from omeroweb import settings` and use the default value of `settings.TOP_LINKS` ?